components/net/http_loader.rs

```diff
@@ -430,9 +430,9 @@ pub fn send_request_to_devtools(
     msg: ChromeToDevtoolsControlMsg,
     devtools_chan: &Sender<DevtoolsControlMsg>,
 ) {
-    devtools_chan
-        .send(DevtoolsControlMsg::FromChrome(msg))
-        .unwrap();
+    if let Err(e) = devtools_chan.send(DevtoolsControlMsg::FromChrome(msg)) {
+        panic!("{e}");
```

Comment:
Replacing .unwrap() with panic!() doesn’t make failure handling more “graceful”, both crash the process. What we want here is either:
- propagate the error to the caller, or
- keep running but add good observability (log + context), or
- if crashing is truly intended, use expect() with a clear, actionable message.

in this case crashing is not needed and we would prefer loging:
```
if let Err(e) = devtools_chan.send(DevtoolsControlMsg::FromChrome(msg)) {
        log::error!("DevTools send failed: {e}");
}
```